gym/f110_gym/envs/rendering.py

```diff
@@ -294,6 +301,42 @@ def on_draw(self):
         self.fps_display.draw()
         # Remove default modelview matrix
         glPopMatrix()
+        
+    def update_waypoints(self, path):
+        """
+        Updates the renderer with the latest waypoint data from the gym environment for the ego vehicle
+        
+        Args:
+            path (array): list of waypoints from path planner
+            
+        Returns:
+            None
+        """
+        
+        # instantiate path w/ set vertex count as Pyglet does not like changing the 
+        # number of vertices for a given polygon after setting it
+        orig_path = [0.]*20
+        
+        if path is not None:
+            np_path = np.array(path)
+            np_path = np_path.flatten() 
+            
+            # add all points from path to orig_path and set leftover x,y values to
+            # be the final value of path
+            for i in range(0,len(orig_path),2):
+                if i >= len(np_path):
+                    orig_path[i] = 50.*np_path[-2]
```

Comment:
Why 50?
Some comments here will be helpful.

Reply:
My bad, I remember the 50 now, it was used as a scaling factor for rendering. I'll add the comment myself.